Client/Frontend/Browser/LoginsHelper.swift

```diff
@@ -184,34 +179,28 @@ class LoginsHelper: TabHelper {
 
         let formatted: String
         if let username = new.username {
-            let promptStringFormat = NSLocalizedString("LoginsHelper.PromptUpdateLogin.Title", value: "Update login %@ for %@?", comment: "Prompt for updating a login. The first parameter is the username for which the password will be updated for. The second parameter is the hostname of the site.")
```

Comment:
Same here. Just creating the Snackbar like you would a UIAlertView. And moved the strings to Strings.swift